tests/Baseline/zeek.file-analyzer-nested/weird.log

```diff
@@ -0,0 +1,11 @@
+### BTest baseline data generated by btest-diff. Do not edit. Use "btest -U/-u" to update. Requires BTest >= 0.63.
+#separator \x09
+#set_separator	,
+#empty_field	(empty)
+#unset_field	-
+#path	weird
+#open XXXX-XX-XX-XX-XX-XX
+#fields	ts	uid	id.orig_h	id.orig_p	id.resp_h	id.resp_p	name	addl	notice	peer	source
+#types	time	string	addr	port	addr	port	string	string	bool	string	string
+1363709111.086948	-	-	-	-	-	file_error	XXXXXXXXXXXXXXXXX: maximal file depth exceeded	F	zeek	-
```

Comment:
Since we do seem to report nothing in this case, do we actually have a chance to report any useful information from upper layers in the file? At least from the parsing side we have some interesting data, even before we hit the recursion limit.

Reply:
Haven't tested it but II think we are actually continuing processing at the upper layer, just the new analyzer aborts directly. I'll see if I can confirm that.

Reply:
Confired, and test extended accordingly.